docs/checkit.js

```diff
@@ -354,12 +356,92 @@ CheckIt.prototype.onAuthStateChanged = function(user) {
     }
 };
 
-// Returns true if user is signed-in. Otherwise false and displays a message.
-CheckIt.prototype.checkSignedInWithMessage = function() {
-    // Return true if the user is signed in Firebase
-    if (this.auth.currentUser) {
-        return true;
-    }
+CheckIt.prototype.generateEmptyCalendar = function(calObj) {
+    // Generate the html for an empty calendar of the calendar you want to 
+    // display.
+    
+    var calendarDiv = this.$calendarDiv;
+    
+    // Add the title of the calendar
+    calendarDiv.append('<div id="calendarTitleHeading"> <h1 class="page-header text-center">' +
+                        calObj.state.title + '</h1></div>');
+                        
+
+    calObj.monthObjects.forEach (function(monthObj, index) {
+
+        var yearHeader = "<div class='page-header text-center'>" +
+                         "<h2 id='yearHeader'>" + monthObj.monthYear + "</h2>" +
```

Comment:
Don't actually need that id, since bootstrap classes take care of the CSS. Removed it.